ts/Extensions/DataGrouping.ts

```diff
@@ -1012,6 +1022,26 @@ seriesProto.generatePoints = function (): void {
     this.groupedData = this.hasGroupedData ? this.points : null;
 };
 
+// When all series are processed, calculate the group pixel width and then
+// if this value is different than zero apply groupings.
+addEvent(Axis, 'postProcessData', function (): void {
+    const axis = this,
+        series = axis.series;
+
+    series.forEach(function (series): void {
+        // Reset the groupPixelWidth, then calculate if needed.
+        series.groupPixelWidth = void 0; // #2110
```

Comment:
Q: I know it's not directly from this PR, but I'm curious if this is enough? Clearing only **current** series `groupPixelWidth`. That means all others still keep old `groupPixelWidth`, isn't it?

Reply:
Hmm... that is an interesting question. 
Theoretically, it seems as that might not be enough however I haven't found any failing scenario for that...

After some thinking, I assume that it should be ok because in the next step the `getGroupPixelWidth` method already knows about all series and if needed this whole loop should set the same `groupPixelWidth` for each series. In this case, it doesn't matter that some series might have old `groupPixelWidth`.